redpanda-standalone/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3.7'
```

Comment:
other files are 3.8, can we make everything use the same version?

Reply:
We discussed, and checked and moved everything to 3.4